desktop/env.js

```diff
@@ -83,5 +85,8 @@ debug( '========================================================================
 // Define a global 'desktop' variable that can be used in browser windows to access config and settings
 global.desktop = {
 	config: config,
-	settings: Settings
+	settings: Settings,
+	editorContextMenu: EditorContextMenu,
+	generalContextMenu: GeneralContextMenu
```

Comment:
I think it's a race condition; we assign to the vars too early. 

This works: https://gist.github.com/mjangda/788bb4654dabbadb030d